packages/@react-spectrum/tabs/test/Tabs.test.js

```diff
@@ -191,12 +191,15 @@ describe('Tabs', function () {
     act(() => {firstItem.focus();});
     expect(firstItem).toHaveAttribute('aria-selected', 'true');
     fireEvent.keyDown(firstItem, {key: 'ArrowRight', code: 39, charCode: 39});
+    fireEvent.keyUp(firstItem, {key: 'ArrowRight', code: 39, charCode: 39});
```

Comment:
firstItem isn't correct, the element that will receive this keyup in a browser is the document.activeElement, or secondItem